src/split_single_dyn_dim.cpp

```diff
@@ -120,6 +150,8 @@ void split_single_dyn_dim::apply(module_pass_manager& mpm) const
         }
         mm->replace_return(outputs);
     }
+
+    migraphx::run_passes(*mm, {migraphx::dead_code_elimination{}});
```

Comment:
You can do `mpm.run_pass(dead_code_elimination{})`, but why dont we just add this pass to target.cpp?